scripts/windows/choco/signalfx-agent/tools/chocolateyinstall.ps1

```diff
@@ -70,28 +68,46 @@ try {
 }
 
 # remove orphaned files or when upgrading from bundle installation
-if (!(msi_installed -name "$env:ChocolateyPackageTitle") -And (Test-Path -Path "$installation_path\SignalFx\SignalFxAgent")) {
+if (!(msi_installed -name "$env:ChocolateyPackageTitle") -And (Test-Path -Path "$install_dir")) {
     # delete symlink first if it exists
-    $link_path = "$installation_path\SignalFx\SignalFxAgent\etc\signalfx"
+    $link_path = "$install_dir\etc\signalfx"
     if ((Test-Path -Path "$link_path") -And ((Get-Item "$link_path").LinkType -eq "SymbolicLink")) {
         Get-Item "$link_path" | %{$_.Delete()}
     }
-    Remove-Item -Recurse -Force "$installation_path\SignalFx\SignalFxAgent" -ErrorAction Ignore
+    Remove-Item -Recurse -Force "$install_dir" -ErrorAction Ignore
+}
+
+$packageArgs = @{
+    packageName    = $env:ChocolateyPackageName
+    fileType       = 'msi'
+    file           = Join-Path "$toolsDir" "MSI_NAME"  # replaced at build time
+    softwareName   = $env:ChocolateyPackageTitle
+    checksum64     = "MSI_HASH"  # replaced at build time
+    checksumType64 = 'sha256'
+    silentArgs     = "/qn /norestart /l*v `"$($env:TEMP)\$($packageName).$($env:chocolateyPackageVersion).MsiInstall.log`" INSTALLDIR=`"$($install_dir)`""
+    validExitCodes = @(0)
 }
 
 Install-ChocolateyInstallPackage @packageArgs
 
 if (!(Test-Path -Path "$config_path")) {
     echo "$config_path not found"
     echo "Copying default agent.yaml to $config_path"
-    Copy-Item "$installation_path\SignalFx\SignalFxAgent\etc\signalfx\agent.yaml" "$config_path"
+    Copy-Item "$install_dir\etc\signalfx\agent.yaml" "$config_path"
 }
 
 echo "Installing agent service..."
-install_service
+install_service -installation_path "$install_dir"
 echo "- Done"
 
-echo "Starting agent service..."
-start_service
-wait_for_service -timeout 60
-echo "- Started"
+if ($access_token -eq "") {
+    echo ""
+    echo "*NOTICE*: To start the SignalFx Smart Agent service, update $program_data_path\token with a valid SignalFx access token and either restart Windows or run the following powershell command:"

```

Comment:
Users have to obtain the access token from the organization to which they want to send data, otherwise the Smart Agent won't work. An access token can be valid for organization "A" and still not work when it's used in organization "B".